src/modules/FileLocksmith/FileLocksmithUI/Views/MainPage.xaml

```diff
@@ -0,0 +1,185 @@
+﻿<Page
+    x:Class="PowerToys.FileLocksmithUI.Views.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:converters="using:PowerToys.FileLocksmithUI.Converters"
+    xmlns:core="using:Microsoft.Xaml.Interactions.Core"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+    xmlns:interop="using:FileLocksmith.Interop"
+    xmlns:labs="using:CommunityToolkit.Labs.WinUI"
+    xmlns:local="using:PowerToys.FileLocksmithUI.Views"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:toolkitConverters="using:CommunityToolkit.WinUI.UI.Converters"
+    mc:Ignorable="d">
+    <Page.Resources>
+        <toolkitConverters:BoolToVisibilityConverter
+            x:Key="boolToVisibilityConverter"
+            FalseValue="Visible"
+            TrueValue="Collapsed" />
+
+        <toolkitConverters:DoubleToVisibilityConverter
+            x:Key="doubleToVisibilityConverter"
+            FalseValue="Visible"
+            GreaterThan="0"
+            TrueValue="Collapsed" />
+        <converters:FileCountConverter x:Key="fileCountConverter" />
+        <converters:PidToIconConverter x:Key="pidToIconConverter" />
+        <converters:PidToUserConverter x:Key="pidToUserConverter" />
+        <converters:FileListToDescriptionConverter x:Key="fileListToDescriptionConverter" />
+    </Page.Resources>
+
+    <interactivity:Interaction.Behaviors>
+        <core:EventTriggerBehavior EventName="Loaded">
+            <core:InvokeCommandAction Command="{x:Bind ViewModel.LoadProcessesCommand}" />
+        </core:EventTriggerBehavior>
+    </interactivity:Interaction.Behaviors>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="48" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <Grid Padding="18,0,18,4" VerticalAlignment="Bottom">
+            <!--<TextBlock Margin="0,2,0,0" Text="Current proccesses:" Style="{ThemeResource BodyStrongTextBlockStyle}"/>-->
+            <TextBlock
+                Margin="0,8,0,0"
+                HorizontalAlignment="Left"
+                Style="{ThemeResource BodyStrongTextBlockStyle}"
+                Text="{x:Bind ViewModel.Paths, Converter={StaticResource fileListToDescriptionConverter}}">
+                <ToolTipService.ToolTip>
+                    <TextBlock TextWrapping="WrapWholeWords" Text="{x:Bind ViewModel.PathsToString, Mode=OneWay}"/>
```

Comment:
Is this tested with long paths containing spaces?
We should bind visibility on the number of paths and show it only with more than 1 path(s).

Reply:
Even with 1 path, it's good to have the tooltip because the path might be too long to show in the UI.

The tooltips seem to not expand, so the previous List would not work well and would trim paths horizontally, making the information kind of useless. I've changed it to a TextBlock with text wrapping to avoid that. Here's what it looks like:
![image](https://user-images.githubusercontent.com/26118718/198262704-e3001264-0ec0-41d6-8086-412f161dc8c8.png)

It still won't show every path if there's lots of files selected, but at least it'll show useful information for long paths.

Reply:
Hmmm. But did you selected all this paths in explorer? What I had in mind was showing the items that the user has selected in explorer window (input parameters). What we have yet is a bit to much.

Reply:
Yes, these were the items I selected. I opened the folder and selected all the files/folders within with select all and only then activated File Locksmith for that selection.